docs/static/breaking-changes-90.asciidoc

```diff
@@ -13,3 +13,34 @@ Include:
 * user impact/decription/value prop
 * link to relevant docs for more information
 
+[discrete]
+[[ssl-settings-9.0]]
+===== Changes to SSL settings in {ls} plugins
+
+We've removed deprecated SSL settings in some {ls} plugins, and have replaced them with updated settings.
+If your plugin configuration contains any of these obsolete options, the plugin may fail to start.
+
+**Plugins with changes to SSL settings** 
+
+[discrete]
+[[output-http-ssl-9.0]]
+.<<plugins-outputs-http,logstash-output-http>>
```

Comment:
```suggestion
.`logstash-output-http`
```

Reply:
The link to the plugin docs added confusion and seems like overkill.  All of the links in the dropdown content take users to the plugin docs anyway.